packages/near-membrane-base/src/membrane.ts

```diff
@@ -611,6 +644,34 @@ export function createMembraneMarshall() {
             return fn;
         }
 
+        function isLiveObject(value: any): boolean {
+            if (value === null || value === undefined || value === ObjectProto) {
+                return false;
+            }
+            if (typeof value === 'object') {
```

Comment:
is this check really needed? I don't think so... if it is just to differentiate functions from objects, then ok.

Reply:
the target could be a function.